### PR TITLE
These were quite handy

### PR DESCRIPTION
**Pull request by errordeveloper**
Hey @erikh, I've used this package and thought you might accept these couple of the small addition...

### CODE REVIEW

**Comment by erikh** [comment]
can you submit these to https://github.com/rendon/testcli instead? This is a fork of that.

**Comment by errordeveloper** [comment]
I've noticed you've added a few features that didn't yet make it upstream, and I got an impressions  upstream may be not maintained very actively...

**Comment by erikh** [comment]
oh, ok. I thought those got merged.

Ok, we'll do this then.

File: main.go [hunk 1]
```diff
@@ -205,3 +213,17 @@ func (c *Cmd) StderrMatches(regex string) bool {
 func StderrMatches(regex string) bool {
 	return pkgCmd.StderrMatches(regex)
 }
+
+func (c *Cmd) StdoutAndStderr() string {
+	return strings.Join(
+		[]string{
```
Comment (erikh) [inline main.go:219]: can we use a struct here instead of magic strings?

**Pull request merged**